app/controllers/bulkrax/importers_controller.rb

```diff
@@ -106,6 +100,22 @@ def external_sets
     end
 
     private
+
+      def files_for_import(file, cloud_files)
+        return if file.blank? && cloud_files.blank?
+        if file.present?
+          @importer[:parser_fields]['import_file_path'] = write_import_file(file)
+        end
+        if cloud_files.present?
+          # For BagIt, there will only be one bag, so we get the file_path back
+          # For Csv, we expect only file uploads, so we won't get the file_path back
+          target = retrieve_files(cloud_files)
+          @importer[:parser_fields]['import_file_path'] = target unless target.blank?
```

Comment:
what happens with @import[:parser_fields]['import_file_path'] is blank on retrieve_files?

Reply:
For CSV we expect import_file_path to already be set - I've improved the comment.